src/client/components/ModuleId.vue

```diff
@@ -1,33 +1,74 @@
 <script setup lang="ts">
-import { computed } from 'vue'
 import { list, root } from '../logic'
 
 const props = withDefaults(
   defineProps<{
     id?: string
     icon?: boolean
+    module?: boolean
   }>(),
   {
     icon: true,
   },
 )
 
 const isVirtual = computed(() => list.value?.modules.find(i => i.id === props.id)?.virtual)
+const moduleName = computed(() =>
+  props.id?.startsWith(root.value)
+    ? props.id.slice(root.value.length)
```

Comment:
we should move the `.` in the template here, the tooltip not showing it (custom span to add `op50`)